pkg/common/default_template.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+var DefaultTemplates = `
```

Comment:
It would be easier to implement this templates with a `map[<provider-name>][]byte`. You don't need to convert twice from yaml to memory, and memory to json. You can just do something like:

```golang
func GetDefaultTemplate(providerName string, out interface{}) error{
  content:=DefaultTemplates[providerName]
  return yaml.Unmarshal(content,out)
}
```

Reply:
fixed